SimTracker/SiPhase2Digitizer/plugins/Phase2TrackerDigitizerAlgorithm.cc

```diff
@@ -45,10 +45,14 @@ namespace {
   constexpr double m_proton = 938.272;
   double calcQ(float x);
 }  // namespace
-namespace {
+namespace { 
   double calcQ(float x) {
```

Comment:
can this function change to return a float (as its calculation is all floating point)?